cucumber-react/test/reducer_test.js

```diff
@@ -1,9 +1,9 @@
 /* eslint-env mocha */
 import assert from "assert"
-import reducer from "../../lib/common/reducer"
+import reducer from "../lib/reducer"
 
 describe(reducer.name, () => {
-  it("keeps a map of sources", () => {
+  it("keeps a map of parsed gherkin AST documents", () => {
```

Comment:
@aslakhellesoy when @tooky and I read the code we had to go looking to be sure that the `sources` in the state contained parsed ASTs and not raw source. Maybe `docs` or `gherkinDocs` would be a clearer name?